metricbeat/magefile.go

```diff
@@ -73,6 +73,11 @@ func CrossBuildGoDaemon() error {
 	return devtools.CrossBuildGoDaemon()
 }
 
+// ModulesCollector collects module configs to modules.d
+func ModulesCollector() error {
+	return metricbeat.CollectModules()
+}
+
```

Comment:
I guess this can be removed too? CI builds are failing because of that.

Reply:
Done, removed.